code/modules/crafting/quality_of_crafting/drying_rack.dm

```diff
@@ -210,6 +210,28 @@
 	craftdiff = 0
 	blacklisted_paths = list(/obj/item/natural/hide/cured)
 
+/datum/repeatable_crafting_recipe/parchment2
+	name = "parchment (timber)"
+	output = /obj/item/paper
+	output_amount = 4
+	starting_atom = /obj/item/weapon/knife
+	tool_usage = list(
+		/obj/item/weapon/knife = list("starts to prepare the log", "start to prepare the log", 'sound/combat/hits/bladed/genstab (1).ogg'),
+	)
+	requirements = list(
+		/obj/item/grown/log/tree = 1,
```

Comment:
this means you can use a large log aswell, just wanted to make sure
but also because subtypes are allowed, this means you can use sticks, since sticks iirc are children from log/tree